lib/matplotlib/pyplot.py

```diff
@@ -2486,24 +2499,26 @@ def getname_val(identifier):
 def _autogen_docstring(base):
     """Autogenerated wrappers will get their docstring from a base function
     with an addendum."""
-    msg = "\n\nAdditional kwargs: hold = [True|False] overrides default hold state"
+    #msg = "\n\nAdditional kwargs: hold = [True|False] overrides default hold state"
+    msg = ''
     addendum = docstring.Appender(msg, '\n\n')
     return lambda func: addendum(docstring.copy_dedent(base)(func))
 
 # This function cannot be generated by boilerplate.py because it may
 # return an image or a line.
 @_autogen_docstring(Axes.spy)
-def spy(Z, precision=0, marker=None, markersize=None, aspect='equal', hold=None, **kwargs):
+def spy(Z, precision=0, marker=None, markersize=None, aspect='equal', **kwargs):
```

Comment:
This is the only place that `hold` in removed a positional argument.  I am mildly :-1: on breaking that.